docs/src/proposals/standalone-tpu-node.md

```diff
@@ -0,0 +1,188 @@
+This also offloads signature verification from the validator.
+
+These design constraints require a new _TPUProxy_ protocol with inbuilt congestion control and authentication.
+It exposes a privileged API that allows registration of TPU endpoints and direct submission of transactions to the banking stage.
+
+The final stage of the TPU node is _Proxy_ which implements the _TPUProxy_ sender.
+The validator receives and processes _TPUProxy_ connections via the _ProxyFetch_ stage.
+
+Validators accept multiple _TPUProxy_ connections to allow for round-robin or ECMP load balancing in the future.
+
+The _ProxyFetch_ recipient should continuously monitor available system resources
+and respond to downstream (banking stage) pressure by throttling the rates of the incoming transactions streams.
+The _Proxy_ stage sender must drop surplus transactions once its network buffers are filled.
+
+```
+     +---------+                                 +------------+
+     | TPU     |                                 | TPUvote    |
+     | Clients |                                 | TPUforward |
+     +----+----+                                 | Clients    |
+          |                                      +-----+------+
+          |                                            |
+          |                                            |
++---------v---------+       +--------------------------v-----------+
+|      TPU node     |       |                       Validator      |
+|                   |       |                                      |
+|   +-----------+   |       |                      +-----------+   |
+|   | Fetch     |   |       |                      | Fetch     |   |
+|   +-----+-----+   |       |                      +-----+-----+   |
+|         |         |       |                            |         |
+|         |         |       |                            |         |
+|   +-----v-----+   |       |   +------------+     +-----v-----+   |
+|   | SigVerify |   |   +-------> ProxyFetch |     | SigVerify |   |
+|   +-----+-----+   |   |   |   +-----+------+     +-----+-----+   |
+|         |         |   |   |         |                  |         |
+|         |         |   |   |         |                  |         |
+|   +-----v-----+   |   |   |         |            +-----v-----+   |
+|   | Proxy     +-------+   |         +------------> Banking   |   |
+|   +-----------+   |       |                      +-----+-----+   |
+|                   |       |                            |         |
++-------------------+       |                            |         |
+                            |                            v         |
+                            |                           ...        |
+                            |                                      |
+                            +--------------------------------------+
+```
+
+The _TPUProxy_ protocol is based on gRPC. Mutual TLS authentication is _mandatory_.
```

Comment:
This is open for discussion. Maybe we want to use QUIC instead

cc @segfaultdoc

Reply:
I suspect it'll come down to latency and perf. If negligible then I lean gRPC ++ protobuf